### DISCUSSION TITLE
Precise Cutting

### DISCUSSION THREAD

**Discussion by MrDeun** [blackears/cyclopsLevelBuilder#146] [Ideas]
I found myself having a decent headache wheneven I try to cut out a pathway from the wall. It would help a lot if there was a tool simillar to clip tool that allows me freely to set as many points as I need so I could easily make for example a hole in the wall for the pipe. Any tips on that?

**Comment by darkhog** [comment]
Which version of Cyclops are you using? AFAIK 1.0.2 has (destructive) CSG operations. Just make a cylinder and subtract it from the wall?
